scripts/travis_script.sh

```diff
@@ -38,6 +38,10 @@ if [[ $GROUP == coverage_and_docs ]]; then
     # Run the coverage check.
     npm run test:coverage
 
+    # Run the link check
+    pip install -q pytest-check-links
+    py.test --check-links **/*.md
```

Comment:
Hmm. Testing further locally, this will only match one level down, not N-levels...